### DISCUSSION TITLE
Quick Reblog "unable to authorize"

### DISCUSSION THREAD

**Discussion by slach123** [AprilSylph/XKit-Rewritten#634] [Help Desk]
![image](https://user-images.githubusercontent.com/95504277/173452293-6a01ab16-40b2-4b8c-96bf-65759214ca20.png)
Every time reblogging since the latest update (0.20.1) (on firefox) it says unable to authorize when I try to use quick reblog to queue, draft, or reblog. I have to move my mouse back down to the reblog button so the menu refreshes??? (Idk) and then it allows me to reblog fine. But when you do that, all the tags you've entered disappear, so it can be quite frustrating.

**Comment by AprilSylph** [comment]
Your Firefox profile is likely broken; if this is consistently happening in your current profile and consistently not happening in a new profile, you'll need to migrate everything over to the new profile. This bug is Tumblr-specific, not XKit-specific.

See also: #206
